Gems/PhysX/Code/Mocks/PhysX/MockSceneInterface.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Interface/Interface.h>
+#include <AzFramework/Physics/PhysicsScene.h>
+#include <gmock/gmock.h>
+
+namespace UnitTest
+{
+    class MockSceneInterface : public AZ::Interface<AzPhysics::SceneInterface>::Registrar
```

Comment:
Similarly to the others, `SceneInterface` is part of AzFramework, so it'd be nice if AzFramework tests project provides it instead of PhysX.

Reply:
I would have to get an approval for that from sig-core who owns AzFramework section. It would have to be a separate PR but it does sound good. I will file a ticket for it.